test/unit/specs/constants.spec.js

```diff
@@ -1,18 +1,41 @@
-let AppleSignin;
+let AppleSignin = require('ti.applesignin');
```

Comment:
This should be a `const` now

```suggestion
const AppleSignin = require('ti.applesignin');
```